assets/global.js

```diff
@@ -542,21 +549,179 @@ class SliderComponent extends HTMLElement {
     } else {
       this.nextButton.removeAttribute('disabled');
     }
+  }
 
-    this.pageCount.textContent = this.currentPage;
-    this.pageTotal.textContent = this.totalPages;
+  onButtonClick(event) {
+    event.preventDefault();
+    this.slideScrollPosition = event.currentTarget.name === 'next' ? this.slider.scrollLeft + this.sliderLastItem.clientWidth : this.slider.scrollLeft - this.sliderLastItem.clientWidth;
```

Comment:
changed this to use `this.slideScrollPosition` because of an issue in Firefox otherwise. 
When trying to reuse the value as a const in the `slideshowComponent` and `super.onButtonClick` it wasn't grabbing the value. Even though that worked on Chrome and Safari.